usr/src/uts/common/fs/zfs/sys/dmu.h

```diff
@@ -350,6 +350,9 @@ typedef struct dmu_buf {
  */
 uint64_t dmu_object_alloc(objset_t *os, dmu_object_type_t ot,
     int blocksize, dmu_object_type_t bonus_type, int bonus_len, dmu_tx_t *tx);
+uint64_t dmu_object_alloc_ibs(objset_t *os, dmu_object_type_t ot, int blocksize,
```

Comment:
i prefer to use prototypes without variables names - only type.
it allow to change variables names in real function without additional update in prototype.

Reply:
Me too.  But I think we should keep a consistent style within each header file.  Would you like to open a PR to remove all the variable names from this file?

Reply:
I think we can do it for `usr/src/uts/common/fs/zfs/sys/*.h`

Reply:
i think it is not related to this review :)
i can start to do it after this upstream - my LGTM for it now, thanks for your comments, Matt.